beacon-chain/types/block.go

```diff
@@ -31,18 +31,7 @@ func NewBlock(data *pb.BeaconBlockResponse) (*Block, error) {
 		return nil, errors.New("invalid block data, parent hash should be 32 bytes")
 	}
 
-	return &Block{data}, nil
-}
-
-// NewGenesisBlock returns the canonical, genesis block for the beacon chain protocol.
-func NewGenesisBlock() (*Block, error) {
```

Comment:
why was this removed ?

Reply:
Yeah, that is what I am wondering and prompted my question for @rawfalafel. How do we get the initial genesis before it's on disk?

Reply:
One client in the network creates the genesis block and broadcasts it to everyone else. Every other client fetches and stores the block during initial sync. Since initial sync isn't done yet, this method should be called in `NewBeaconChain`, right next to active/crystallized state initialization.